mobilecoind/clients/java/mob_client/src/main/proto/mobilecoind_api.proto

```diff
@@ -339,18 +341,18 @@ message GenerateTxRequest {
 
     // List of UnspentTxOuts to be spent by the transaction.
     // All UnspentTxOuts must belong to the same sender_monitor_id.
-    // Mobilecoind would choose a subset of these inputs to construct the transaction.
+    // mobilecoind would choose a subset of these inputs to construct the transaction.
     // Total input amount must be >= sum of outlays + fees.
     repeated UnspentTxOut input_list = 3;
 
     // Outputs to be generated by the transaction. This excludes change and fee.
     repeated Outlay outlay_list = 4;
 
-    // Fee (optional, setting to 0 would cause mobilecoind to choose a fee).
+    // Fee (setting to 0 causes mobilecoind to choose a fee).
     uint64 fee = 5;
 
-    // Tombstone block to use for the transaction. Note that this can later be changed by manipulating
-    // tx_proposal.tx.tombstoneb_clock
+    // Tombstone block to use for the transaction. This can be changed later by manipulating
```

Comment:
Same here - tombstone block can't actually be changed.

Reply:
yeah, this file is meant to be a duplicate. my understanding is that it could eventually be a symlink depending on the java build.